pylhc_submitter/job_submitter.py

```diff
@@ -273,214 +326,18 @@ def main(opt):
     else:
         LOG.info("Starting Job-submitter.")
 
-    opt = _check_opts(opt)
-    save_config(opt.working_directory, opt, "job_submitter")
-
-    job_df = _create_jobs(
-        opt.working_directory,
-        opt.mask,
-        opt.jobid_mask,
-        opt.replace_dict,
-        opt.job_output_dir,
-        opt.append_jobs,
-        opt.executable,
-        opt.script_arguments,
-        opt.script_extension,
-    )
-    job_df, dropped_jobs = _drop_already_ran_jobs(
-        job_df, opt.resume_jobs or opt.append_jobs, opt.job_output_dir, opt.check_files
-    )
+    save_config(Path(opt.working_directory), opt, "job_submitter")
+    creation_opt, runner_opt = check_opts(opt)

```

Comment:
Why are we saving the config before checking options now?

Reply:
This is on purpose. Two reasons:
a) I want the ini-file as close to the user-input as possible (even with mistakes).
b) the `check_opts` does not give the checked opts back, as it transforms it into the creation/runner options. So they are not even available for saving anymore

Reply:
Okay. That second point escaped me, I'm fine with all this.